client/reducers/events.js

```diff
@@ -67,11 +68,19 @@ const eventsReducer = (state=initialState, action) => {
             return {
                 ...state,
                 showEventDetails: action.payload,
+                showEventForm: true,
             }
         case 'CLOSE_EVENT_DETAILS':
             return {
                 ...state,
                 showEventDetails: null,
+                showEventForm: false,
+            }
+        case 'DELETE_EVENT':
+            return {
+                ...state,
+                showEventDetails: action.payload,
```

Comment:
why updating `showEventDetails` and `showEventForm` here ? I find a little bit confusing `showEventDetails` and `showEventForm`. Maybe one of them can be removed or renamed because the names suggest to me the same thing.